src/Lib/Managers/ModeManager.vala

```diff
@@ -127,6 +127,18 @@ public class Akira.Lib.Managers.ModeManager : Object {
         }
     }
 
+    /*
+     * Returns optional extra context. See InteractionMode for more details.
+     */
+    public Object? active_mode_extra_context () {
+        if (pan_mode != null) {
+            return pan_mode.extra_context ();
+        }
+
+        return (active_mode != null) ? active_mode.extra_context () : null;
```

Comment:
I think the parenthesis are not required for a ternary condition.

Reply:
It isn't but I think it's better for readability.But I changed it.